utility/importer/brave_external_process_importer_bridge.h

```diff
@@ -8,12 +8,17 @@
 
 #include <string>
 
+#include "brave/common/importer/brave_importer_bridge.h"
+#include "brave/common/importer/profile_import.mojom.h"
 #include "chrome/utility/importer/external_process_importer_bridge.h"
 
-class BraveExternalProcessImporterBridge
-    : public ExternalProcessImporterBridge {
+class BraveExternalProcessImporterBridge : public ExternalProcessImporterBridge,
```

Comment:
following https://github.com/brave/brave-core/pull/5582/commits/0494a09ac517dbbea74a7fda9170916ac1985599#r441903894
we don't need to inherit `ExternalProcessImporterBridge` if we run different importer process

Reply:
I think reusing `ExternalProcessImporterBridge` is good because we still use whole `chrome::mojom::ProfileImportObserver` interface and `brave::mojom:ProfileImportObserve` only defines our new options.

Reply:
ok, then you just need to deal with https://github.com/brave/brave-core/pull/5582/files#r441900049

Reply:
Thanks updated.